### PR TITLE
📝(a11y) adding a11y docs

### PR DESCRIPTION
**Pull request by Leimi**
Help the richie team and community contribute in a more accessible way with this doc

### CODE REVIEW

**Comment by Leimi** [comment]
Hey @sampaccoud, @jbpenrath, what do you think?

File: accessibility.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+Note that if the progress is described as text in addition to the bar (like, "60%" written next to the bar), it's not mandatory to make the bar colors accessible. Because the information itself is now accessible through the text.
+
+**tl;dr: every color necessary for the use and understanding of a component must be contrasted enough with what it touches**
+
+## Text alternatives
+
+When it comes to textual alternatives for images, there are several things to keep in mind.
+
+## Only images that carry information
```
Comment (jbpenrath) [inline accessibility.md:41]:
```suggestion
### Only images that carry information
```

**Review by jbpenrath — COMMENTED** [review]
Can you add a summary on the top of the document to access quickly to a section of this doc ?

- Colors
- Text alternatives
- Links
- Headings
- Dom elements order
- Client side navigation and keyboard management
- Forms

File: accessibility.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
+# Notable points about accessibility
+
+These are a few accessibility (a11y) related things to keep in mind when touching HTML/CSS/JS code in Richie.
+
+Note that these are not "accessibility basics", since by the looks of the code in richie, they are already well-known within the team. These are things to know to go the extra step when caring about a11y, that I most often noticed while contributing on Richie in 2022.
+
+Of course not everything is set in stone: there are always exceptions related to each use case, and often multiple ways to solve a11y issues. So, remember this is not about unconditionally following rules. It's about making sure everyone has access to all information on your website, and can interact with it. So, when in doubt: test! Accessibility panels in browser devtools can help as a first checkup, but the best is to actually use screen readers. NVDA + Firefox and VoiceOver + Safari are common setups that you can use to make sure what you ship actually works.
```
[the remark this answers is not in the document]
Comment (Leimi) [reply accessibility.md]: good idea, done, thanks

File: accessibility.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+
+- [Colors](#colors)
+- [Text alternatives](#text-alternatives)
+- [Links](#links)
+- [Headings](#headings)
+- [DOM elements order](#dom-elements-order)
+- [Client side navigation and keyboard management](#client-side-navigation-and-keyboard-management)
+- [Forms](#forms)
+
+These are a few accessibility (a11y) related things to keep in mind when touching HTML/CSS/JS code in Richie.
```
Comment (jbpenrath) [inline accessibility.md:13]: This doc does not target only Richie.
```suggestion
These are a few accessibility (a11y) related things to keep in mind when touching HTML/CSS/JS code.
```

**Review by jbpenrath — APPROVED** [review]
A very useful guide to keep on hand 👏

**Pull request merged**